mypy/nodes.py

```diff
@@ -697,7 +697,8 @@ def set_line(self,
             arg.set_line(self.line, self.column, self.end_line)
 
     def is_dynamic(self) -> bool:
-        return self.type is None
+        from mypy.types import CallableType
```

Comment:
how come this import isnt at the top of the file? if its because of a circular import can it be avoided in a better way?

Reply:
I don't think so, `types` depends on `nodes` a lot, idk how to pull it out.